ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/DeliveryDestinationsForm.tsx

```diff
@@ -139,7 +151,38 @@ function DeliveryDestinationsForm({
                         <FlexItem>
                             Configure or setup a schedule to share reports on a recurring basis.
                         </FlexItem>
-                        <FlexItem flex={{ default: 'flexNone' }} />
+                        <FlexItem flex={{ default: 'flexNone' }}>
+                            <Flex direction={{ default: 'row' }}>
+                                <FlexItem>
+                                    <RepeatScheduleDropdown
+                                        label="Repeat every"
+                                        isRequired
+                                        fieldId="schedule.intervalType"
+                                        value={formValues.schedule.intervalType || ''}
+                                        handleSelect={onScheduledRepeatChange}
+                                    />
+                                </FlexItem>
+                                <FlexItem>
+                                    <DayPickerDropdown
+                                        label="On day(s)"
+                                        isRequired
+                                        fieldId={
+                                            formValues.schedule.intervalType === 'WEEKLY'
+                                                ? 'schedule.daysOfWeek'
+                                                : 'schedule.daysOfMonth'
+                                        }
+                                        value={
+                                            formValues.schedule.intervalType === 'WEEKLY'
+                                                ? formValues?.schedule?.daysOfWeek || []
+                                                : formValues?.schedule?.daysOfMonth || []
```

Comment:
Nit: Same here, aren't `formValues`, `formValues.schedule` and `formValues.schedule.daysOfWeek` all non-nullable?